hetmech/test_matrix.py

```diff
@@ -15,53 +59,16 @@ def test_metaedge_to_adjacency_matrix():
         'test/data/disease-gene-example-graph.json',
     )
     graph = hetio.readwrite.read_graph(url)
+    row_names, col_names, adj_mat = \
+        metaedge_to_adjacency_matrix(graph, test_edge, matrix_type=mat_type,
+                                     dtype=dtype)
+    exp_row, exp_col, exp_adj = get_arrays(test_edge, mat_type, dtype)
 
-    # Verify GiG matrix
-    gig_rows = ['CXCR4', 'IL2RA', 'IRF1', 'IRF8', 'ITCH', 'STAT3', 'SUMO1']
-    gig_cols = ['CXCR4', 'IL2RA', 'IRF1', 'IRF8', 'ITCH', 'STAT3', 'SUMO1']
-
-    row_names, col_names, adj_mat = metaedge_to_adjacency_matrix(graph, 'GiG')
-    assert np.array_equal(row_names, gig_rows)
-    assert np.array_equal(col_names, gig_cols)
-    assert np.array_equal(adj_mat, [[0, 0, 1, 0, 1, 0, 0],
-                                    [0, 0, 1, 0, 0, 0, 0],
-                                    [1, 1, 0, 1, 0, 0, 1],
-                                    [0, 0, 1, 0, 0, 0, 0],
-                                    [1, 0, 0, 0, 0, 0, 0],
-                                    [0, 0, 0, 0, 0, 0, 0],
-                                    [0, 0, 1, 0, 0, 0, 0]])
-
-    # Verify GaD matrix
-    gad_rows = ['CXCR4', 'IL2RA', 'IRF1', 'IRF8', 'ITCH', 'STAT3', 'SUMO1']
-    gad_cols = ["Crohn's Disease", 'Multiple Sclerosis']
-
-    row_names, col_names, adj_mat = metaedge_to_adjacency_matrix(graph, 'GaD')
-    assert np.array_equal(row_names, gad_rows)
-    assert np.array_equal(col_names, gad_cols)
-    assert np.array_equal(adj_mat, [[0, 1],
-                                    [0, 1],
-                                    [1, 0],
-                                    [0, 1],
-                                    [0, 0],
-                                    [1, 1],
-                                    [0, 0]])
-
-    # Verify DlT matrix
-    dlt_rows = ["Crohn's Disease", 'Multiple Sclerosis']
-    dlt_cols = ['Leukocyte', 'Lung']
-
-    row_names, col_names, adj_mat = metaedge_to_adjacency_matrix(graph, 'DlT')
-    assert np.array_equal(row_names, dlt_rows)
-    assert np.array_equal(col_names, dlt_cols)
-    assert np.array_equal(adj_mat, [[0, 0],
-                                    [1, 0]])
-
-    # Verify TlD matrix
-    tld_rows = ['Leukocyte', 'Lung']
-    tld_cols = ["Crohn's Disease", 'Multiple Sclerosis']
-
-    row_names, col_names, adj_mat = metaedge_to_adjacency_matrix(graph, "TlD")
-    assert np.array_equal(row_names, tld_rows)
-    assert np.array_equal(col_names, tld_cols)
-    assert np.array_equal(adj_mat, [[0, 1],
-                                    [0, 0]])
+    assert row_names == exp_row
+    assert col_names == exp_col
+    assert type(adj_mat) == type(exp_adj)  # Arrays same type
+    assert not sparse.csc_matrix(adj_mat != exp_adj).nnz  # Arrays equal
```

Comment:
From [`numpy.array_equal`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.array_equal.html), "True if two arrays have the same shape and elements, False otherwise."

So let's do the same

```
assert adj_mat.shape == exp_adj.shape
assert (adj_mat == exp_adj).all()
```

Assuming this works